unittests/PhasarLLVM/DataFlow/IfdsIde/Problems/IDELinearConstantAnalysisSwiftTest.cpp

```diff
@@ -81,17 +83,583 @@ class IDELinearConstantAnalysisSwiftTest : public ::testing::Test {
+}
+
+TEST_F(IDELinearConstantAnalysisSwiftTest, HandleCallTest_08) {
+  auto Results = doAnalysis("call_08.ll");
+  std::set<LCACompactResult_t> GroundTruth;
+  GroundTruth.emplace("$s7call_086MyMainV3fooyS2i_SitFZ", 4, "a", 10);
+  GroundTruth.emplace("$s7call_086MyMainV3fooyS2i_SitFZ", 4, "b", 1);
+
+  GroundTruth.emplace("$s7call_086MyMainV4mainyyFZ", 8, "i", 10);
+  GroundTruth.emplace("$s7call_086MyMainV4mainyyFZ", 9, "i", 10);
+  GroundTruth.emplace("$s7call_086MyMainV4mainyyFZ", 9, "j", 1);
+  GroundTruth.emplace("$s7call_086MyMainV4mainyyFZ", 11, "i", 10);
+  GroundTruth.emplace("$s7call_086MyMainV4mainyyFZ", 11, "j", 1);
+
+  compareResults(Results, GroundTruth);
+}
+
+TEST_F(IDELinearConstantAnalysisSwiftTest, HandleCallTest_09) {
+  auto Results = doAnalysis("call_09.ll");
+  std::set<LCACompactResult_t> GroundTruth;
+
+  GroundTruth.emplace("$s7call_096MyMainV4mainyyFZ", 8, "i", 43);
+  GroundTruth.emplace("$s7call_096MyMainV4mainyyFZ", 9, "i", 43);
+  GroundTruth.emplace("$s7call_096MyMainV4mainyyFZ", 9, "j", 43);
+
+  compareResults(Results, GroundTruth);
+}
+
+TEST_F(IDELinearConstantAnalysisSwiftTest, HandleCallTest_10) {
+  auto Results = doAnalysis("call_10.ll");
+  std::set<LCACompactResult_t> GroundTruth;
+  GroundTruth.emplace("$s7call_106MyMainV3baryySiFZ", 3, "b", 2);
+  GroundTruth.emplace("$s7call_106MyMainV3fooyySiFZ", 8, "a", 2);
+
+  compareResults(Results, GroundTruth);
+}
+
+TEST_F(IDELinearConstantAnalysisSwiftTest, HandleCallTest_11) {
+  auto Results = doAnalysis("call_11.ll");
+  std::set<LCACompactResult_t> GroundTruth;
+  GroundTruth.emplace("$s7call_116MyMainV3baryS2iFZ", 4, "b", 2);
+
+  GroundTruth.emplace("$s7call_116MyMainV3fooyS2iFZ", 8, "a", 2);
+
+  GroundTruth.emplace("$s7call_116MyMainV4mainyyFZ", 13, "i", 2);
+  compareResults(Results, GroundTruth);
+}
+
+/* ============== LOOP TESTS ============== */
+TEST_F(IDELinearConstantAnalysisSwiftTest, HandleLoopTest_01) {
+  auto Results = doAnalysis("while_01.ll");
+  std::set<LCACompactResult_t> GroundTruth;
+  GroundTruth.emplace("$s8while_016MyMainV4mainyyFZ", 4, "i", 42);
+  compareResults(Results, GroundTruth);
+}
+
+TEST_F(IDELinearConstantAnalysisSwiftTest, HandleLoopTest_03) {
+  auto Results = doAnalysis("while_03.ll");
+  std::set<LCACompactResult_t> GroundTruth;
+  GroundTruth.emplace("$s8while_036MyMainV4mainyyFZ", 4, "i", 42);
+  GroundTruth.emplace("$s8while_036MyMainV4mainyyFZ", 9, "a", 13);
+  compareResults(Results, GroundTruth);
+}
+
+TEST_F(IDELinearConstantAnalysisSwiftTest, HandleLoopTest_04) {
+  auto Results = doAnalysis("while_04.ll");
+  std::set<LCACompactResult_t> GroundTruth;
+  GroundTruth.emplace("$s8while_046MyMainV4mainyyFZ", 4, "i", 42);
+  GroundTruth.emplace("$s8while_046MyMainV4mainyyFZ", 7, "a", 0);
+  compareResults(Results, GroundTruth);
+}
+
+/* ============== Global TESTS ============== */
+
+// TEST_F(IDELinearConstantAnalysisSwiftTest, HandleGlobalsTest_01) {
```

Comment:
You may want to enable the commented code below and mark the tests as disabled by starting their name with `DISABLED_` or by calling the macro `GTEST_SKIP()` from within the test, e.g. 
```C++
GTEST_SKIP() << "Swift-globals are not supported yet";
```